.github/workflows/terraform-plan.yml

```diff
@@ -0,0 +1,115 @@
+name: Terraform plan
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+    - 'config/clusters/**'
+
+defaults:
+  run:
+    working-directory: config/clusters
+
+permissions:
+  pull-requests: write
+
+env:
```

Comment:
I don't love for a PR CI to have write access to the cluster (using eg: https://github.com/falcosecurity/test-infra/blob/master/.github/workflows/master.yml#L21).
Do we really need this?

Reply:
Write actually is not needed, we should setup a tailored IAM Role.
The authentication instead is needed to generate the plan